src/sdk/Audio/AudioInputStream.ts

```diff
@@ -188,7 +188,7 @@ export class PushAudioInputStreamImpl extends PushAudioInputStream implements IA
             const readCycle = (): Promise<Blob | Buffer> => {
                 return audioNode.read().then<Blob | Buffer>((audioStreamChunk: IStreamChunk<ArrayBuffer>) => {
                     if (!audioStreamChunk || audioStreamChunk.isEnd) {
-                        if (typeof (XMLHttpRequest) !== "undefined") {
+                        if (typeof (XMLHttpRequest) !== "undefined" && typeof (Blob) !== "undefined") {
```

Comment:
Are you sure that it is necessary to check `XMLHttpRequest` here at all? On line 197 only `Blob` is checked.

Reply:
Yes. There is an alternate case in Node where Blob is defined but XMLHttpRequest is not, and a Buffer type must be returned then.